File: unittest/scripts/js_devapi/scripts/dba_cluster_no_misconfigurations_interactive.js
// Assumptions: smart deployment routines available
//@ Initialization
testutil.deploySandbox(__mysql_sandbox_port1, "root", {report_host: hostname});
testutil.snapshotSandboxConf(__mysql_sandbox_port1);

//@ connect
shell.connect({scheme:'mysql', host: localhost, port: __mysql_sandbox_port1, user: 'root', password: 'root'});

//@<OUT> create cluster no misconfiguration: ok
var cluster = dba.createCluster('dev', {memberSslMode:'REQUIRED'});

session.close();

//@ Finalization
testutil.destroySandbox(__mysql_sandbox_port1);

File: unittest/scripts/py_devapi/scripts/dba_cluster_no_misconfigurations_interactive.py
# Assumptions: smart deployment rountines available
#@ Initialization
deployed_here = reset_or_deploy_sandboxes();

#@ connect
shell.connect({'scheme': 'mysql', 'scheme': 'mysql', 'host': localhost, 'port': __mysql_sandbox_port1, 'user': 'root', 'password': 'root'})

#@<OUT> create cluster no misconfiguration: ok
if __have_ssl:
  cluster = dba.create_cluster('dev', {'memberSslMode':'REQUIRED'})
else:
  cluster = dba.create_cluster('dev')

session.close()

#@ Finalization
# Will delete the sandboxes ONLY if this test was executed standalone
if (deployed_here):
  cleanup_sandboxes(True)
